spec/lib/pending_assigns_reminder_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe PendingAssignsReminder do
+  let(:user) { Fabricate(:user) }
+
+  it { assert_reminder_not_created }
+
+  it 'Do not create a reminder if the user only has one task' do
+    post = Fabricate(:post)
+    TopicAssigner.new(post.topic, user).assign(user)
+
+    assert_reminder_not_created
+  end
+
+  def assert_reminder_not_created
```

Comment:
This assertion doesn't feel correct. What if `post_created` event is never called? Perhaps we can just assert that no new post has been created?